src/openrct2/GameStateSnapshots.cpp

```diff
@@ -551,7 +550,7 @@ struct GameStateSnapshots final : public IGameStateSnapshots
             const rct_sprite& spriteCmp = spritesCmp[i];
 
             changeData.spriteIdentifier = spriteBase.generic.sprite_identifier;
-            changeData.miscIdentifier = spriteBase.generic.type;
+            changeData.miscIdentifier = spriteBase.generic.SubType;
```

Comment:
This will put nonsense in this field for non misc entities but the value is only used when it is a misc entity so it should be fine.

Reply:
Maybe we should comment about that in the code?